tests/phpunit/tests/rest-api/rest-attachments-controller.php

```diff
@@ -566,6 +566,51 @@ public function test_get_items_valid_date() {
 		$this->assertEquals( $id2, $data[0]['id'] );
 	}
 
+	public function test_get_items_invalid_modified_date() {
```

Comment:
```suggestion
	/*
	 * @ticket 50617
	 */
	public function test_get_items_invalid_modified_date() {
```

`@ticket` annotations should be added for any new tests introduced.

Reply:
Added the doc block to all new tests.